addon.xml

```diff
@@ -1,15 +1,15 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
-<addon id="plugin.video.twitch" version="2.5.4" name="Twitch" provider-name="anxdpanic, A Talented Community">
+<addon id="plugin.video.twitch" version="2.5.4+matrix.1" name="Twitch" provider-name="anxdpanic, A Talented Community">
     <requires>
-        <import addon="xbmc.python" version="2.20.0"/>
+        <import addon="xbmc.python" version="3.0.0"/>
         <import addon="script.module.six" version="1.11.0"/>
         <import addon="script.module.python.twitch" version="2.0.16"/>
         <import addon="script.module.requests" version="2.9.1"/>
     </requires>
     <extension point="xbmc.python.pluginsource" library="resources/lib/addon_runner.py">
         <provides>video</provides>
     </extension>
-    <extension point="xbmc.service" library="resources/lib/service_runner.py" start="login"/>
+    <extension point="xbmc.service" library="resources/lib/service_runner.py"/>
```

Comment:
Please remove this from the PR

Reply:
I've done it. Looks like, It's incorrect fork. I did that without Git.